demisto_sdk/commands/common/tests/readme_test.py

```diff
@@ -192,3 +196,22 @@ def test_verify_no_default_sections_left(integration, capsys, file_input, sectio
     section_error = f'Replace "{section}" with a suitable info.'
     assert not result
     assert section_error in stdout
+
+
+def test_invalid_short_file(capsys):
+    """
+    Given
+        - Read me that contains sections that are created as default and need to be changed
```

Comment:
Need to fix this docstring to correspond to the length and not to the sections (I guess its just a copy-paste error)
```suggestion
        - Readme that contains sections that are created as default and need to be changed
```